src/AsyncTCP.h

```diff
@@ -37,8 +37,8 @@ extern "C" {
 
 //If core is not defined, then we are running in Arduino or PIO
 #ifndef CONFIG_ASYNC_TCP_RUNNING_CORE
-#define CONFIG_ASYNC_TCP_RUNNING_CORE -1 //any available core
-#define CONFIG_ASYNC_TCP_USE_WDT 1 //if enabled, adds between 33us and 200us per event
+#define CONFIG_ASYNC_TCP_RUNNING_CORE -1 // Any available core, but sticking to one core is recommended if using SPIFFS/LittleFS.
```

Comment:
Why is it necessary to confine the AsyncTCPSock task to one core when using flash filesystem (SPIFFS/LittleFS) access? Do you have examples where this causes problems and pinning to a core fixes it?

Reply:
The issue occurred within my project when I was pushing the cores and RAM to their limits by serving very large files from LittleFS while running several tasks that didn't leave a lot of time to the scheduler. I'm not entirely sure where the issue is(no debug probe), but it caused the async_tcp task to time out without a panic.